<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MechanicalMaterial</class>
 <widget class="QWidget" name="MechanicalMaterial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mechanical material</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="GroupQuantities_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1677215</height>
      </size>
     </property>
     <property name="title">
      <string>Material</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QComboBox" name="cb_materials">
        <item>
         <property name="text">
          <string>choose...</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="l_mat_description">
        <property name="text">
         <string>Material Description</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupQuantities_4">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1677215</height>
      </size>
     </property>
     <property name="title">
      <string>Properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Poisson Ratio:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Young's Modulus:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="spinBox_poisson_ratio">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.300000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="Gui::InputField" name="input_fd_young_modulus">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="unit" stdset="0">
           <string notr="true">Pa</string>
          </property>
          <property name="decimals" stdset="0">
           <number>3</number>
          </property>
          <property name="maximum" stdset="0">
           <double>2000000000.000000000000000</double>
          </property>
          <property name="singleStep" stdset="0">
           <double>2.000000000000000</double>
          </property>
          <property name="value" stdset="0">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Shapes of the model</string>
     </property>
     <layout class="QGridLayout" name="_2">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="cb_material_shapes">
        <item>
         <property name="text">
          <string>all</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>remaining</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>referenced</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QStackedWidget" name="widget_stack">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_all">
         <layout class="QGridLayout" name="_3">
          <property name="margin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="2" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="l_all">
            <property name="text">
             <string>all</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_remaining">
         <layout class="QGridLayout" name="_5">
          <property name="margin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="l_remaining">
            <property name="text">
             <string>remaining</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>51</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_referenced">
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="1" column="0">
           <widget class="QListWidget" name="list_References"/>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_Reference">
            <property name="text">
             <string>Add reference</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupQuantities_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1677215</height>
      </size>
     </property>
     <property name="title">
      <string>External material resources</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QPushButton" name="pushButton_MatWeb">
        <property name="text">
         <string>MatWeb database...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::InputField</class>
   <extends>QLineEdit</extends>
   <header>Gui/InputField.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cb_material_shapes</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>widget_stack</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
